lib/ht/search_client/test.rb

```diff
@@ -1,14 +1,14 @@
 module Ht::SearchClient::Test
 
   refine Ht::SearchClient::Remote.singleton_class do
-    def stub_request(params = {})
-      new(params).stub_request
+    def stub_request(params = {}, method = :get)
+      new(params).stub_request(method)
     end
   end
 
   refine Ht::SearchClient::Remote do
-    def stub_request
-      @webmock = Ht::SearchClient::Connection.stub_request(endpoint, params)
+    def stub_request(method)
+      @webmock = Ht::SearchClient::Connection.stub_request(endpoint, params, method)
```

Comment:
do you think here is feasible to do something like
```
@webmock = Ht::SearchClient::Connection.stub_request(endpoint, params, self.method)
```

This way it will never be exposed that is a POST ?

What do you think ?